app/controllers/organisation/consultations_controller.rb

```diff
@@ -135,6 +135,19 @@ def invite_respondents
     redirect_back fallback_location: root_path,  flash_success_info: "Respondent was successfully invited."
   end
 
+  def import_responses
+    if params[:consultation][:file].present?
+      import = ConsultationResponse.import_responses(params[:consultation][:file])
+      if import == false
```

Comment:
@BALAJI-BALA - if the import is a success what does it return? Perhaps we could return a count of the number of records that were imported.

Reply:
@BALAJI-BALA - I suggest that the import method returns a Hash, with the status and also the number of records, then we can check for success rather than failure first. That would also make the code a bit more optimistic haha